pandas/_libs/tslibs/util.pxd

```diff
@@ -2,23 +2,13 @@
 from cpython cimport PyTypeObject
 
 cdef extern from *:
-    """
-    PyObject* char_to_string(const char* data) {
-    #if PY_VERSION_HEX >= 0x03000000
```

Comment:
I think there's one more PY_VERSION_HEX check in compat_helper that could go

Reply:
Yea I was looking at that but I'm planning on the compat stuff in a  separate PR